LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Adel Ghabboun
```

Comment:
Just wanted to check if there are any legal issues related to this. Does this need to reference Microsoft? Like

` Copyright (c) Microsoft Corporation.`

Reply:
This was auto generated so I am not sure about it. I will look at the old samples and maybe @paulomarquesc can chime in

Reply:
you are right. changed it